src/CMakeLists.txt

```diff
@@ -57,6 +60,7 @@ set(freerct_SRCS ${freerct_SRCS}
     "${CMAKE_SOURCE_DIR}/src/generated/coasters_strings.h"
     "${CMAKE_SOURCE_DIR}/src/generated/gui_strings.h"
     "${CMAKE_SOURCE_DIR}/src/generated/shops_strings.h"
+    "${CMAKE_SOURCE_DIR}/src/generated/comp_info.h"
```

Comment:
maybe compile_info.h (and .cpp? the nuances of how constant strings work still confuse me)
rev == revision, fyi ;)